client/components/GlobalControls/GlobalControlsButton.tsx

```diff
@@ -55,10 +55,14 @@ const GlobalControlsButton = React.forwardRef((props: Props, ref: React.Ref<unkn
 			large: !isMobileButton,
 			className: classNames(className, renderProps.className),
 		};
+
+		// ? need to tell props that it will be buttonprops explicitly
```

Comment:
I have this question mark here but feel there is a better way to express the two props. Also not sure why I need to update this as the only bp package that needed an upgrade is bp select